README.md

```diff
@@ -1,119 +1,61 @@
- # Arc Enabled Kubernetes Validation Program 
+ # Azure Arc Validation Program 
 
- Azure Arc aims at bringing management and services to any infrastructure. Deploy and manage applications on Kubernetes clusters everywhere using GitOps.  
+While Azure Arc can be deployed on any K8s clusters, the Arc validation program provides distinct visibility for integrated K8s platforms and promises quality, support and consistency for Azure Arc customers across different hybrid clusters. 
```

Comment:
'promises' is going too far as we can never provide 100% guarantee, suggest rephrasing this sentence

Reply:
Addressed this